web/client/themes/default/less/geostory.less

```diff
@@ -76,7 +76,9 @@
         }
     }
     .ms-cascade-story.ms-edit {
-        .ms-section-title, .ms-section-immersive {
+        .ms-section-title,
+        .ms-section-banner,
```

Comment:
I think this style is not needed because it applies to content style `.ms-text-editor-active`